.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1325,6 +1340,36 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             return ret;
         }
 
+        // Parse entities that are made up by two time points with now
+        private DateTimeParseResult ParseNowAsDate(string text, DateObject referenceDate)

```

Comment:
What happens if there are two "now" in the sentence?
For example: "When I say now, I mean now!". Can you also add that as a spec case?

Reply:
We will match "between..and.." in stage Extract, so if just have two "now" and not have "between/from" , the text would not be extracted.
I will add the example as a spec case.

Reply:
But the method usage is not clear. Even the code description.
What about a phrase like "Let's meet between now and Friday; not right now, ok?"?
Please also add it in specs.